---
topology_marker_ns: topology_graph
show_block_outlines: true
use_gvd_block_outlines: false
gvd_visualizer:
  visualization_type: 2    # 0 -> NONE, 1 -> ESDF_SLICE, 2 -> GVD
  color_nearest_vertices: false
  gvd_alpha: 0.6
  gvd_min_distance: 0.2
  gvd_max_distance: 3.0
  basis_threshold: 2
  min_num_basis: 1
  max_num_basis: 10
  gvd_mode: 0    # 0 -> DEFAULT, 1 -> DISTANCE, 2 -> BASIS_POINTs
  esdf_alpha: 0.6
  slice_height: 1.0
  esdf_min_distance: 0.2
  esdf_max_distance: 5.0
graph_visualizer:
  z_offset_scale: 0.0
  visualize: true
  marker_scale: 0.2
  marker_alpha: 0.9
  use_sphere_marker: true
  use_label: true
  label_height: 0.7
  label_scale: 0.3
  use_bounding_box: false
  bounding_box_alpha: 0.5
  use_edge_source: false
  interlayer_edge_scale: 0.1
  interlayer_edge_alpha: 0.4
  interlayer_edge_use_color: true
  interlayer_edge_insertion_skip: 0
  intralayer_edge_scale: 0.02
  intralayer_edge_alpha: 0.4
  intralayer_edge_use_color: false
  intralayer_edge_insertion_skip: 0
visualizer_colormap:
  min_hue: 0.0
  max_hue: 0.5
  min_saturation: 0.9
  max_saturation: 0.9
  min_luminance: 0.6
  max_luminance: 0.85
